### PR TITLE
Add Scaleway setup

### PR DESCRIPTION
**Pull request by demangejeremy**
### Description

Add Scaleway setup in documentation.
#1913

### CODE REVIEW

**Comment by demangejeremy** [comment]
Capture d’écran 2022-09-03 à 05 36 17

CLA well signed but not validated :/

File: docs/guides/scaleway-setup.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Examples
+
+### Set the Metastore URI
+
+In your [node config file](/docs/configuration/node-config), use `metastore_uri: s3://{your-bucket}/{your-indexes}`.
+
+### Set the Index URI
+
+In your [index config file](/docs/configuration/index-config), use `index_uri: s3://{your-bucket}/{your-indexes}`
```
Comment (fulmicoton) [inline docs/guides/scaleway-setup.md:41]:
```suggestion
In your [index config file](/docs/configuration/index-config), use `index_uri: s3://{your-bucket}/{your-indexes}`.
```

**Comment by fulmicoton** [comment]
I do see your username in the list on CLA Assistant, so I confirm you have signed the CLA. I tried to push an extra commit to retrigger the CLA check. That did not work unfortunately.

**Comment by fulmicoton** [comment]
@demangejeremy Ah I found the problem.. You need to set your email on git and use it in your commit.

[add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: docs/guides/scaleway-setup.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Declare the environment variables used by Quickwit to configure the storage:
+```bash
+export AWS_ACCESS_KEY_ID=****
+export AWS_SECRET_ACCESS_KEY=****
+```
+   
+2. Set the endpoint URI: 
+```bash
+export QW_S3_ENDPOINT=https://s3.fr-par.scw.cloud
```
Comment (maximkpa) [inline docs/guides/scaleway-setup.md:22]:
```suggestion
export QW_S3_ENDPOINT=https://s3.{your-region}.scw.cloud
```

File: docs/guides/scaleway-setup.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+3. Set the region: 
+```bash
+export AWS_REGION=fr-par
```
Comment (maximkpa) [inline docs/guides/scaleway-setup.md:27]:
```suggestion
export AWS_REGION={your-region}
```

File: docs/guides/scaleway-setup.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+### Set the Metastore URI
+
+In your [node config file](/docs/configuration/node-config), use `metastore_uri: s3://{your-bucket}/{your-indexes}`.
+
+### Set the Index URI
+
+In your [index config file](/docs/configuration/index-config), use `index_uri: s3://{your-bucket}/{your-indexes}`.
+
+:::note
+Note that the URI scheme has still the name `s3` but Quickwit is actually sending HTTP requests to `https://s3.fr-par.scw.cloud`.
```
Comment (maximkpa) [inline docs/guides/scaleway-setup.md:44]:
```suggestion
Note that the URI scheme has still the name `s3` but Quickwit is actually sending HTTP requests to `https://s3.{your-region}.scw.cloud`.
```

File: docs/guides/scaleway-setup.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+title: Object Storage (Scaleway) setup
```
Comment (maximkpa) [inline docs/guides/scaleway-setup.md:2]:
```suggestion
title: Scaleway (object storage) setup
```

**Review by maximkpa — COMMENTED** [review]
The guide looks amazing, thank you for the help! I thought of just keeping region broad here for the endpoint, but other than that all good.

**Review by fmassot — APPROVED** [review]

**Comment by demangejeremy** [comment]
Problem with CLA Assistant is resolved :)

**Comment by fmassot** [comment]
thanks @demangejeremy !

**Pull request merged**